go-controller/pkg/kubevirt/dhcp.go

```diff
@@ -104,17 +145,15 @@ func ComposeDHCPv4Options(cidr, dnsServer, controllerName string, vmKey ktypes.N
 	return composeDHCPOptions(controllerName, vmKey, dhcpOptions)
 }
 
-func ComposeDHCPv6Options(cidr, dnsServer, controllerName string, vmKey ktypes.NamespacedName) *nbdb.DHCPOptions {
+func ComposeDHCPv6Options(cidr, controllerName string, vmKey ktypes.NamespacedName) *nbdb.DHCPOptions {
 	serverMAC := util.IPAddrToHWAddr(net.ParseIP(ARPProxyIPv6)).String()
 	dhcpOptions := &nbdb.DHCPOptions{
 		Cidr: cidr,
 		Options: map[string]string{
 			"server_id": serverMAC,
+			"fqdn":      fmt.Sprintf("%q", vmKey.Name),
```

Comment:
your commit msg omits you're also setting the hostname for ipv6. 

Maybe consider exporting this to a separate commit ? Also, shouldn't it be an option ? Or are we after ipv4 parity (which always sets the hostname).

Reply:
> your commit msg omits you're also setting the hostname for ipv6.

Sure let me add it to the commit message

> 
> Maybe consider exporting this to a separate commit ? Also, shouldn't it be an option ? Or are we after ipv4 parity (which always sets the hostname).

Idea is to have IPv4 parity, so singlestack IPv6 works as expected, there is no special option because hypershift and primary UDN should do the same here, don't want to add complexity if not needed.